### ISSUE TITLE
Hey man 8)

### ISSUE THREAD

**Issue by alanhu1996** [jksison/Personal#1]

**Comment by jksison** [comment]
Hey Alan! What's up?

**Comment by alanhu1996** [comment]
My phone got wiped :p can you hit me up with your number again?

**Comment by jksison** [comment]
hey did you change your number?

**Comment by alanhu1996** [comment]
i'm in the US atm but i will be using my old number when i get back haha i don't have access to that number right now though

**Comment by jksison** [comment]
Nice, it looks like you work there. when will you be back?

**Comment by alanhu1996** [comment]
Yep! Internship ends late dec, so I'll be back in Canada by then haha. Got school right after in Jan though!

**Comment by jksison** [comment]
That's pretty cool. What projects are you working on?

**Comment by alanhu1996** [comment]
Doing some react and ruby stuff haha hey are you on any other platform? (facebook etc)

**Comment by jksison** [comment]
Cool, you're a react and ruby master now. maybe you can teach me when you are back. 

i deleted my facebook account.

**Comment by alanhu1996** [comment]
Haha not a master :D u got to teach me the ways man

**Comment by jksison** [comment]
it looks like you are on github almost everyday. what's the algos for and where do you get the source code from?
